plugins/modules/system/sudoers.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+# Copyright: (c) 2019, Jon Ellis (@JonEllis) <ellis.jp@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: sudoers
+short_description: manage sudoers files
```

Comment:
```suggestion
short_description: Manage sudoers files
```

Reply:
This is undoing the case change requested by @russoz in https://github.com/ansible-collections/community.general/pull/3746#discussion_r750897699.

I'm happy either way, just let me know how it's supposed to be.

Reply:
We had this discussion already in another PR ;) While the guidelines say that it should be lower-case, right now it's almost everywhere upper-case, so for consistency let's do it upper-case. (We can change it at some point in a mass-edit.)

Reply:
Fair enough. I'll go for upper case first character then.